src/pages/timecard/Timecard.spec.jsx

```diff
@@ -80,6 +80,48 @@ describe('Shift spanning mutiple days', () => {
     });
   });
 
+  it('should display start and finish time on timecard when both times have been entered', async () => {
+    renderWithTimecardContext(<Timecard />, {
+      summaryErrors: {},
+      setSummaryErrors: jest.fn(),
+      timeEntries: [
+        {
+          timePeriodTypeId: '00000000-0000-0000-0000-000000000001',
+          startTime: '2022-09-01 01:00:00+00:00',
+          finishTime: '2022-09-03 05:00:00+00:00',
```

Comment:
Assuming this test is actually specifically to cover those covering multiple days, it might be better to test the boundaries of the logic/requirement by having the start time be 11:59 and the finish time 00:00 the following day.

Reply:
that's been changed now